int16 id
geometry_msgs/Point position
SensorArray role
float64             operation_range
float64             approx_param
float64             smoke_variance
float64             camera_range
float64             angle_of_view
float64             camera_variance